projects/kit/components/arrow/arrow.component.ts

```diff
@@ -5,7 +5,9 @@ import {
     TuiHostedDropdownComponent,
     TuiTextfieldSizeDirective,
 } from '@taiga-ui/core';
-import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
+import {PolymorpheusComponent, PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+import {ArrowOptions,TUI_ARROW_OPTIONS} from './arrow-options';
```

Comment:
No space after comma. That is weird, prettier should have taken care of that in precommit 🤔 @splincode any idea why this could have happened?

Reply:
I have no idea why, very strange, @mullinsmikey could you figure out what went wrong?

Reply:
Sure. I'll look into this issue either in a couple hours